dcmdata/libsrc/dcrleccd.cc

```diff
@@ -465,11 +466,12 @@ OFCondition DcmRLECodecDecoder::decodeFrame(
 
     DcmPixelItem *pixItem = NULL;
     Uint8 * rleData = NULL;
-    const size_t bytesPerStripe = imageColumns * imageRows;
+    const size_t bytesPerStripe = static_cast<Uint32>(imageColumns) * static_cast<Uint32>(imageRows);
```

Comment:
I think you should use static_cast<size_t> instead of staic_cast<Uint32> here.

bytesPerStripe result is size_t.